### ISSUE TITLE
Window log on launch

### ISSUE THREAD

**Issue by keith** [br1sk/brisk#127]
When launching brisk there is this log message:

```
Unknown Window class (null) in Interface Builder file,
	 creating generic Window instead
```

This appears to be a SDK bug http://www.openradar.me/35511761 https://stackoverflow.com/questions/47051682/unknown-window-class-null-in-interface-builder-file-creating-generic-window-i

**Comment by keith** [comment]
This is fixed in Xcode 9.3 beta 1

**Comment by keith** [comment]
Released in 1.2.0 https://github.com/br1sk/brisk/releases
